src/main/java/com/google/devtools/build/lib/runtime/BuildSummaryStatsModule.java

```diff
@@ -60,6 +63,10 @@
   private long executionEndMillis;
   private SpawnStats spawnStats;
   private Path profilePath;
+  private static final long UNKNOWN = -1;
+  private long cpuUserTimeForActions = 0;
```

Comment:
Please document the behaviour of these fields WRT the "UNKNOWN" value - as I read the code, if any action does not have CPU time, the field becomes UNKNOWN and no other CPU time values are recorded.